config/config_models.go

```diff
@@ -1,58 +1,106 @@
 package config
 
-import "time"
+import (
+	"io/ioutil"
+	"path/filepath"
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/rs/zerolog/log"
+	"gopkg.in/yaml.v2"
+)
+
+// Config general framework config
+type Config struct {
+	FrameworkConfig    *FrameworkConfig    `envconfig:"FRAMEWORK_CONFIG_FILE" default:"../framework.yaml"`
+	NetworksConfig     *NetworksConfig     `envconfig:"NETWORKS_CONFIG_FILE" default:"../networks.yaml"`
```

Comment:
I propose the default path for any config to be `suite` or `e2e` directory, it's how it is in most of our repos